docs/sphinx/_toc.yml.in

```diff
@@ -1,30 +1,31 @@
-# Anywhere {branch} is used, the branch name will be substituted.
-# These comments will also be removed.
+---
+defaults:
+  numbered: false
 root: index
 subtrees:
-  - caption: How to Guides
+  - entries:
+      - file: what-is-hipFORT.rst
+        title: What is hipFORT?
+  - caption: Install
     entries:
-    - file: how_to_guides/getting_started
-  - caption: Reference
+      - file: install/quick-start.rst
+        title: Quick start
```

Comment:
```suggestion
        title: Quick start installation
```

Reply:
@lpaoletti We have the following titles on the portal:

- Quick start installation guide
- Quick-start
- Quick start install guide

Should we add 'guide' to Quick start installation? If not, this would be a fourth variation. Please advise.

Reply:
Let's good with the first one
3 is my limit on variations ;)

Reply:
@Rmalavally  - comment above

Reply:
Sounds good, Leo.